ansible/library/generate_golden_config_db.py

```diff
@@ -116,13 +153,18 @@ def generate_smartswitch_golden_config_db(self):
         return json.dumps(gold_config_db, indent=4)
 
     def generate(self):
+        # topo check
         if self.topo_name == "mx" or "m0" in self.topo_name:
             config = self.generate_mgfx_golden_config_db()
         elif self.topo_name == "t1-28-lag":
             config = self.generate_smartswitch_golden_config_db()
         else:
             config = "{}"
 
+        # version check
+        if self.check_bmp_version() is True:
+            config = self.generate_bmp_golden_config_db(config)
```

Comment:
There are cases that config didn't have FEATURE table. Then the FEATURE table would have only one entry bmp which result in failure. 

In that case, we should generate FEATURE table from init_cfg.json, e.g.:https://github.com/sonic-net/sonic-mgmt/blob/a77166ca7f258e8e744563b8e716432c7ba7f862/ansible/library/generate_golden_config_db.py#L63-L65

Reply:
fixed.